.github/workflows/cache-performance-tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Cache Performance Tests
+
+on:
+  pull_request:
+    branches: ['main']
+    paths:
+      - 'src/Core/Customer/Infrastructure/Repository/CachedCustomerRepository.php'
+      - 'src/Shared/Infrastructure/Cache/**'
+      - 'src/Core/Customer/Application/EventSubscriber/*CacheInvalidation*.php'
+      - 'config/packages/cache.yaml'
+      - 'tests/Integration/Customer/Infrastructure/Repository/CachePerformanceTest.php'
+      - 'tests/Load/scripts/rest-api/cachePerformance.js'
+      - 'tests/Load/execute-load-test.sh'
+      - 'tests/Load/config.json.dist'
+
+jobs:
+  cache-integration-tests:
+    name: Cache Integration Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ vars.PHP_VERSION }}
+
+      - name: Install Dependencies
+        run: composer install -q --no-ansi --no-interaction --no-scripts --no-progress --prefer-dist
```

Comment:
We don't need to run composer install, because make start should install composer dependencies under the hood